### ISSUE TITLE
Driver cannot display date nor timestamp

### ISSUE THREAD

**Issue by diequeiroz** [dacort/metabase-athena-driver#38]
When I try to display date related fields (date or timestamp) the driver cannot display those fields.

<img width="1111" alt="Screen Shot 2020-04-23 at 15 43 41" src="https://user-images.githubusercontent.com/6049847/80137223-57df2400-8579-11ea-8150-774c203ba9e2.png">

Anyone experiencing this same problem?

**Comment by dacort** [comment]
Oh dear thanks @diequeiroz - I’ll try to take a look at this this weekend. Which version of Metabase are you using?

**Comment by diequeiroz** [comment]
@dacort I'm using the docker image metabase/metabase:v0.35.2

**Comment by dacort** [comment]
This looks very similar to #27 but I resolved that with [v0.3.0](https://github.com/dacort/metabase-athena-driver/releases/tag/v0.3.0) release. 

@diequeiroz Can you confirm which version of my driver you're using?

I'm unable to reproduce this with Metabase v0.35.2 and v1.0.0 of my driver.

Query:

```sql
SELECT
    CAST('2019-11-16 18:22:09.097' AS TIMESTAMP) AS timestamp,
    CAST('2019-11-16' AS DATE) AS date,
    CAST('22:12' AS TIME) as time,
    true as boolean
```

Screenshot:
![image](https://user-images.githubusercontent.com/1512/80321150-be5d7f80-87cf-11ea-94e8-00b10c95ee73.png)

**Comment by dacort** [comment]
Closing as can't reproduce/no other info provided. Please comment if you're still experiencing this on the most recent version.
